README.md

```diff
@@ -320,3 +334,5 @@ Postgres Converter provides the following functionality
 converts into Metadata object 
 
 - **generate_to_meta:** (function) takes the database connection and returns a list of Metadata object for all the (non-system schemas) schemas and tables from the connection.
+
+NOTE: You should be using the SQLAlchemy converter with a postgres sqlalchemy connection.
```

Comment:
slightly confused by this note, likely due to my lack of experience with posgres. Does this mean "if you're connecting to postgres with sqlalchemy (rather than psycopg2), you should use the SQLAlchemy converter instead"? Or does it mean "you shouldn't use the postgres converter, because it's better to connect to postgres with sqlalchemy and therefore you should use the SQLAlchemy converter instead"? I'm guessing it's the former, but one could reasonably read it either way. I think there's scope for expanding on why there's a separate postgres converter.

Some options for how you might want to pitch it:
1. "The postgres converter converts metadata got with the psycopg2 package, which is the most common way of connecting to postgres, but if you wanted to connect via sqlalchemy you need to use the sqlalchemy converter"
2. "It is generally recommended to connect to postgres via sqlalchemy, for which we have a sqlalchemy converter, but you may prefer psycopg2 for feature A/B/C in which case use the postgres converter"
3. "the sqlalchemy converter is more robust and should be the default method for most databases, but the postgres converter is retained for compatibility"

The true line is likely "we made postgres converter first, then made sqlalchemy converter which largely supersedes it" but that's not necessarily something you need to know.

Reply:
updated